lib/Sema/CSFix.h

```diff
@@ -251,6 +251,10 @@ enum class FixKind : uint8_t {
 
   /// A warning fix that allows a coercion to perform a force-cast.
   AllowCoercionToForceCast,
+  
+  /// Allow key path root type mismatch when applying a key path with a 
```

Comment:
...applying a key path **to** a root type...

Reply:
Is actually `with` meaning a key path that has a root type not convertible to the base :)

Reply:
I see, you could put it that way.